pkg/api/vlabs/validate.go

```diff
@@ -13,10 +13,25 @@ func (o *OrchestratorProfile) Validate() error {
 	switch o.OrchestratorType {
 	case DCOS:
 	case DCOS190:
+		if e := o.ValidateOrchestratorVersion(DCOS190Version); e != nil {
+			return e
+		}
 	case DCOS188:
+		if e := o.ValidateOrchestratorVersion(DCOS188Version); e != nil {
+			return e
+		}
 	case DCOS187:
+		if e := o.ValidateOrchestratorVersion(DCOS187Version); e != nil {
+			return e
+		}
 	case DCOS184:
+		if e := o.ValidateOrchestratorVersion(DCOS184Version); e != nil {
+			return e
+		}
 	case DCOS173:
+		if e := o.ValidateOrchestratorVersion(DCOS173Version); e != nil {
+			return e
+		}
 	case Swarm:
 	case Kubernetes:
```

Comment:
Its not needed. This check is to make sure that DCOS orchestrator and version is aligned.